test/test_ledger.py

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env python3
+
+import argparse
+import atexit
+import json
+import os
+import socket
+import subprocess
+import time
+import unittest
+
+from bitcoinrpc.authproxy import AuthServiceProxy, JSONRPCException
+from test_device import DeviceTestCase, start_bitcoind, TestDeviceConnect, TestDisplayAddress, TestGetKeypool, TestSignTx
+
+from hwilib.commands import process_commands
+
+def ledger_test_suite(emulator, rpc, userpass):
+    # Look for real ledger using HWI API(self-referential, but no other way)
+    enum_res = process_commands(['enumerate'])
+    path = None
+    master_xpub = None
+    fingerprint = None
+    for device in enum_res:
+        if device['type'] == 'ledger':
+            fingerprint = device['fingerprint']
+            path = device['path']
+            master_xpub = process_commands(['-f', fingerprint, 'getmasterxpub'])['xpub']
+            break
+    assert(path is not None and master_xpub is not None and fingerprint is not None)
+
+    class LedgerTestCase(unittest.TestCase):
```

Comment:
`LedgerTestCase` is unnecessary as there are no device specific tests for the Ledger.